com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Parser/HrJobRoles.php

```diff
@@ -148,7 +148,11 @@ function summary(&$values) {
     $session = CRM_Core_Session::singleton();
     $dateType = $session->get('dateTypes');
 
-    $contractDetails = CRM_Hrjobcontract_BAO_HRJobContract::checkContract($params['job_contract_id']);
+    $contractDetails = NULL;
```

Comment:
as I said in another replay I will squash "PR review suggestions" commit , but these changes were required to be able to run the tests .